app/src/components/game.js

```diff
@@ -1,32 +1,97 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import '../App.css';
 import io from 'socket.io-client';
 import 'react-chessground/dist/styles/chessground.css';
 import Chess from 'chess.js';
+import '../chess.css';
 
-const Chessground = require('chessground').Chessground;
+const { Chessground } = require('chessground');
 
 const socket = io('http://localhost:5010');
+
 const Game = () => {
   const chess = new Chess();
+  let ground;
+
+  const [inCheck, setInCheck] = useState(false);
+  const [inCheckMate, setInCheckMate] = useState(false);
+  const [inDraw, setInDraw] = useState(false);
+  const [config, updateConfig] = useState({
```

Comment:
Jag tänker att det här state objektet är för komplext. Jättebra modellering av schack-modellen men att skicka in hela objektet med metoder till state kan skapa problem för react längre fram(svårt att räkna ut ett changeset). Kan vi göra state objektet enklare och behålla den här modellen på annan plats?